src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length--31 calendar--start-day--fri">
```

Comment:
its should be modifiers, when i change for example so
```suggestion
    <div class="calendar calendar--month-length-29 calendar--start-day-tue">
```
It should have 29 days and first cell should be on Tuesday

Reply:
прочитай нормально завдання